### ISSUE TITLE
Simplified Derivation of the Cross-Entropy Loss Gradient (for a kid)

### ISSUE THREAD

**Issue by sakshamlakhera** [COD1995/ml-meta#10]
In this PR, i have tried to explain cross entropy loss and it's gradient calculation in oder to make it even understandable to a kid.

**Comment by sakshamlakhera** [comment]
How the page looks like:

<img width="1505" alt="Image" src="https://github.com/user-attachments/assets/744d5dd3-64b1-4c79-bb4a-2796625533b3" />
